src/index.html

```diff
@@ -3,12 +3,26 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="index.css">
     <title>Leaderboard</title>
 </head>
 <body>
     <div id="leaderboard">
-        <h1> Leaderboard!</h1>
+        <h1> Highscore Leaderboard </h1>
+    </div>
+    <div class="score-container">
+        <div class="recent-scores">
+            <h2>Recent scores</h2>
+            <button class="refresh">Refresh</button>
+        </div>
+        <div class="add-score">
+            <h2>Add your score</h2>
+            <form class="container">
+                <input type="text" id="name" placeholder="Add your name..." autocomplete="off">
+                <input type="number" id="score" placeholder="Add your score..." autocomplete="off">
+                <button class="add">Add score</button>
+            </form>
+        </div>
     </div>
 </body>
 </html>
```

Comment:
The UI should look like the provided wireframe design. However, in your case, the scores container is missing. Kindly add the same. 
- Optional. Consider populating some scores manually.

| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://github.com/Arch-Noize/leaderboard-API/assets/33268927/d62c8fd4-2376-4b0b-bb8e-30ab297d6396) | ![image](https://github.com/Arch-Noize/leaderboard-API/assets/33268927/49a9f703-5ebd-4253-8661-649d3fa488aa) |